consistency/analysis/clustering.ipynb

```diff
@@ -0,0 +1,412 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipy \n",
+    "import numpy as np\n",
+    "import strsimpy \n",
+    "from Bio import pairwise2\n",
+    "from Bio.Seq import Seq \n",
+    "from Bio.pairwise2 import format_alignment \n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from sklearn.cluster import dbscan\n",
+    "from strsimpy.levenshtein import Levenshtein\n",
+    "from strsimpy.normalized_levenshtein import NormalizedLevenshtein\n",
+    "from strsimpy.jaro_winkler import JaroWinkler\n",
+    "from strsimpy.longest_common_subsequence import LongestCommonSubsequence\n",
+    "from strsimpy.metric_lcs import MetricLCS\n",
+    "from strsimpy.ngram import NGram\n",
+    "from strsimpy.sorensen_dice import SorensenDice\n",
+    "from strsimpy import SIFT4\n",
+    "from sklearn.metrics.cluster import homogeneity_score\n",
+    "from sklearn.metrics.cluster import completeness_score \n",
+    "from sklearn.metrics.cluster import v_measure_score\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "levenshtein = Levenshtein()\n",
+    "normalized_levenshtein = NormalizedLevenshtein()\n",
+    "lcs = LongestCommonSubsequence()\n",
+    "metric_lcs = MetricLCS()\n",
+    "fourgram = NGram(4)\n",
+    "dice = SorensenDice(2)\n",
+    "s = SIFT4()\n",
+    "jarowinkler = JaroWinkler()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "tagged_df = pd.read_csv (r'/home/gelaw/work-stuff/gocode/src/registry-experimental/consistency/rpc/google/cloud/apigeeregistry/v1/similarity/algorithms /vocab1000.csv')\n",
```

Comment:
I missed this somehow. You might want to take out these absolute paths and replace with relative path so that anyone running this notebook should be able to read the csv successfully.